Makefile

```diff
@@ -429,6 +429,34 @@ build-archive:
 release-unix: clean full build-archive
 	@if [ -f e/Makefile ]; then $(MAKE) -C e release; fi
 
+# Set MacOS/Darwin cert vars for signing/notarizing
+# TSH_SKELETON is a directory name relative to build.assets/macos/
+ENVIRONMENT_NAME ?= promote
+
+# TODO(camh): Verify key names for production keys
+DEVELOPER_KEY_NAME_promote = Developer ID Application: Ada Lin
+INSTALLER_KEY_NAME_promote = Developer ID Installer: Ada Lin
+TEAMID_promote = QH8AA5B8UP
+TELEPORT_BUNDLEID_promote = com.gravitational.teleport
+TSH_BUNDLEID_promote = $(TEAMID).com.gravitational.teleport.tsh
+TSH_SKELETON_promote = tsh
+
+DEVELOPER_KEY_NAME_build = Developer ID Application: Ada Lin
+INSTALLER_KEY_NAME_build = Developer ID Installer: Ada Lin
+TEAMID_build = K497G57PDJ
+TELEPORT_BUNDLEID_build = com.goteleport.dev
+TSH_BUNDLEID_build = $(TEAMID).com.goteleport.tshdev
```

Comment:
Should we keep these in the Makefile, or would it make more sense to keep these as variables/secrets in Github environments (if that's a thing) or something?

Reply:
Looks like they were set in the other script before so I guess it's ok.

Reply:
Walt did not want to put them in the github environment, and they were in the script before, so I went with that. Also putting them in the environment makes it harder to run the commands locally for testing - you still need these vars somewhere for that.